--- ob_display.c.orig	2018-01-14 15:18:34.481476198 +0300
+++ ob_display.c	2018-01-14 15:19:27.766475116 +0300
@@ -44,7 +44,7 @@
 
 		g_string_append_printf (context->builder,
 		    "<menu id=\"openbox-%s\" label=\"%s\" icon=\"%s\">\n",
-		    dir_id, dir_name, dir_icon);
+		    dir_id, dir_id, dir_icon);
 		g_free (dir_icon);
 	}
 	else
@@ -52,7 +52,7 @@
 	{
 		g_string_append_printf (context->builder,
 	      "<menu id=\"openbox-%s\" label=\"%s\">\n",
-	      dir_id, dir_name);
+	      dir_id, dir_id);
 	}
 
 	g_free (dir_id);
