--- foobnix-3.1.01.orig/debian/README.source
+++ foobnix-3.1.01/debian/README.source
@@ -0,0 +1,2 @@
+
+See /usr/share/doc/quilt/README.source
--- foobnix-3.1.01.orig/debian/changelog
+++ foobnix-3.1.01/debian/changelog
@@ -0,0 +1,188 @@
+foobnix (3.1.01-dmo1) unstable; urgency=medium
+
+  * New upstream release.
+  * Simplify 01_converter-local-ffmpeg patch.
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Sat, 06 Jun 2015 15:52:54 +0200
+
+foobnix (3.0.04-dmo1) unstable; urgency=medium
+
+  * New upstream bugfix release.
+  * Move gstreamer1.0-plugins-bad from Recommends to Depends (Really
+    needed).
+  * Add mate-notification-daemon | xfce4-notifyd to Suggests.
+  * Add ffmpeg in Depends (Needed for encoding) and add a patch to disable
+    the download of a "special" ffmpeg binary.
+  * Note : This package is ugly, files are still installed in
+    /usr/lib/python2.7/
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Fri, 13 Mar 2015 13:18:19 +0100
+
+foobnix (3.0.03-dmo2) unstable; urgency=medium
+
+  * Should Depends on python-dbus
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Tue, 03 Mar 2015 08:59:39 +0100
+
+foobnix (3.0.03-dmo1) unstable; urgency=medium
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Tue, 03 Mar 2015 08:41:56 +0100
+
+foobnix (2.6.10-dmo1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Mon, 25 Mar 2013 17:40:23 +0100
+
+foobnix (2.6.09-dmo2) unstable; urgency=low
+
+  * Need to depends on python-webkit.
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Fri, 08 Mar 2013 15:02:18 +0100
+
+foobnix (2.6.09-dmo1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Fri, 08 Mar 2013 14:41:56 +0100
+
+foobnix (2.6.03-dmo1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Wed, 10 Oct 2012 08:38:42 +0200
+
+foobnix (2.6.02-dmo1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Tue, 09 Oct 2012 08:35:45 +0200
+
+foobnix (2.6.01-dmo1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Sun, 02 Sep 2012 08:37:36 +0200
+
+foobnix (2.5.36-dmo2) unstable; urgency=low
+
+  * Build for python >= 2.7
+
+ -- Christian Marillat <marillat@deb-multimedia.org>  Sun, 20 May 2012 10:11:02 +0200
+
+foobnix (2.5.36-dmo1) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Thu, 03 May 2012 08:42:55 +0200
+
+foobnix (2.5.35-0.0) unstable; urgency=low
+
+  * New upstream release.
+  * Switch to dh_python2
+
+ -- Christian Marillat <marillat@debian.org>  Thu, 05 Jan 2012 09:53:17 +0100
+
+foobnix (2.5.34-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Sun, 01 Jan 2012 15:27:29 +0100
+
+foobnix (2.5.33-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Sun, 25 Dec 2011 10:27:36 +0100
+
+foobnix (2.5.32-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Mon, 19 Dec 2011 08:42:07 +0100
+
+foobnix (2.5.30-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Fri, 16 Dec 2011 08:34:05 +0100
+
+foobnix (2.5.29-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Sun, 11 Dec 2011 09:33:04 +0100
+
+foobnix (2.5.28-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Wed, 02 Nov 2011 15:15:40 +0100
+
+foobnix (2.5.26-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Sun, 18 Sep 2011 08:44:19 +0200
+
+foobnix (2.5.25-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Mon, 12 Sep 2011 08:36:53 +0200
+
+foobnix (2.5.23-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Wed, 10 Aug 2011 09:22:48 +0200
+
+foobnix (2.5.22-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Sat, 06 Aug 2011 09:10:06 +0200
+
+foobnix (2.5.20-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Tue, 02 Aug 2011 08:36:34 +0200
+
+foobnix (2.5.19-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Tue, 12 Jul 2011 09:15:43 +0200
+
+foobnix (2.5.18-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Sat, 09 Jul 2011 09:03:45 +0200
+
+foobnix (2.5.17-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Tue, 05 Jul 2011 08:48:41 +0200
+
+foobnix (2.5.16o-0.0) unstable; urgency=low
+
+  * New upstream release.
+
+ -- Christian Marillat <marillat@debian.org>  Sat, 25 Jun 2011 18:36:04 +0200
+
+foobnix (0.2.5-0.0) unstable; urgency=low
+
+  * Upload to my repository.
+
+ -- Christian Marillat <marillat@debian.org>  Mon, 07 Mar 2011 07:06:04 +0100
+
+foobnix (0.2.5-7m) maverick; urgency=low
+
+  * Initial release (Closes: #10007)  Upload new release 0.2.5-7m
+  
+ -- Ivan Ivanenko <ivan.ivanenko@gmail.com>  Tue, 01 Mar 2011 00:23:06 +0200
--- foobnix-3.1.01.orig/debian/compat
+++ foobnix-3.1.01/debian/compat
@@ -0,0 +1 @@
+9
--- foobnix-3.1.01.orig/debian/control
+++ foobnix-3.1.01/debian/control
@@ -0,0 +1,19 @@
+Source: foobnix
+Section: sound
+Priority: extra
+Maintainer: Christian Marillat <marillat@deb-multimedia.org>
+Bugs: mailto:marillat@deb-multimedia.org
+Homepage: http://www.foobnix.com
+X-Python3-Version: >= 3.3
+Standards-Version: 3.9.6
+Build-Depends: debhelper (>= 9), dh-python, quilt,
+  python | python-all | python-dev | python-all-dev
+
+Package: foobnix
+Architecture: all
+Depends: ${python:Depends}, ${misc:Depends}, python-dbus, python-chardet, python-gi, python-simplejson, python-mutagen, gstreamer1.0-plugins-good, gir1.2-gstreamer-1.0, gir1.2-gst-plugins-base-1.0, gir1.2-gtk-3.0, gir1.2-webkit-3.0, gir1.2-soup-2.4, gir1.2-notify-0.7, gir1.2-soup-2.4, gettext, gstreamer1.0-plugins-bad, ffmpeg
+Recommends: python-setuptools, gir1.2-keybinder-3.0, gstreamer1.0-plugins-ugly, gstreamer1.0-libav, gstreamer1.0-alsa, fuseiso, libmp3lame0, libfaac0
+Suggests: mate-notification-daemon | xfce4-notifyd
+Description: Simple and powerful music player for Linux.
+ All the best features in one player. Written in Python, Foobnix is a small,
+ fast, customizable, powerful music player with a user-friendly interface.
--- foobnix-3.1.01.orig/debian/copyright
+++ foobnix-3.1.01/debian/copyright
@@ -0,0 +1,39 @@
+This package was debianized by:
+
+    Ivan Ivanenko <ivan.ivanenko@gmail.com> on Thu, 18 Nov 2010 16:57:35 +0200
+
+It was downloaded from:
+
+    http://www.foobnix.com
+
+Upstream Author:
+
+    Ivan Ivanenko <ivan.ivanenko@gmail.com>
+
+Copyright:
+
+    Ivan Ivanenko <ivan.ivanenko@gmail.com>
+
+License:
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This package is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+On Debian systems, the complete text of the GNU General
+Public License version 3 can be found in `/usr/share/common-licenses/GPL-3'.
+
+The Debian packaging is:
+
+    Copyright © 2010 Ivan Ivanenko <ivan.ivanenko@gmail.com>
+
+and is licensed under the GPL version 3, see above.
--- foobnix-3.1.01.orig/debian/docs
+++ foobnix-3.1.01/debian/docs
@@ -0,0 +1 @@
+README
--- foobnix-3.1.01.orig/debian/patches/01_converter-local-ffmpeg.diff
+++ foobnix-3.1.01/debian/patches/01_converter-local-ffmpeg.diff
@@ -0,0 +1,24 @@
+--- a/foobnix/util/converter.py
++++ b/foobnix/util/converter.py
+@@ -16,7 +16,6 @@ from gi.repository import Gtk
+ from gi.repository import GLib
+ from subprocess import Popen, PIPE
+ 
+-from foobnix.fc.fc_helper import CONFIG_DIR
+ from foobnix.util.const import ICON_FOOBNIX
+ from foobnix.util.file_utils import open_in_filemanager
+ from foobnix.util.localization import foobnix_localization
+@@ -27,11 +26,8 @@ from foobnix.gui.service.path_service im
+ foobnix_localization()
+ 
+ LOGO = get_foobnix_resourse_path_by_name(ICON_FOOBNIX)
+-FFMPEG_NAME = "ffmpeg_foobnix"
+-#fix win
+-if os.name == 'posix':
+-    if os.uname()[4] == 'x86_64':
+-        FFMPEG_NAME += "_x64"
++FFMPEG_NAME = "ffmpeg"
++CONFIG_DIR = "/usr/bin/"
+ 
+ class Converter(ChildTopWindow):
+     def __init__(self):
--- foobnix-3.1.01.orig/debian/patches/series
+++ foobnix-3.1.01/debian/patches/series
@@ -0,0 +1 @@
+01_converter-local-ffmpeg.diff
--- foobnix-3.1.01.orig/debian/rules
+++ foobnix-3.1.01/debian/rules
@@ -0,0 +1,21 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --with quilt,python2
+
+override_dh_auto_build:
+	python setup.py build
+
+override_dh_auto_install:
+	python setup.py install --prefix=$(CURDIR)/debian/foobnix/usr --install-layout=deb
+
+override_dh_auto_clean:
+	python setup.py clean
+
+override_dh_clean:
+	dh_clean foobnix/__init__.pyc foobnix/version.pyc test/__init__.pyc \
+	test/all.pyc
+	rm -rf dist
+
+override_dh_builddeb:
+	dh_builddeb -- -Zbzip2
--- foobnix-3.1.01.orig/debian/source/format
+++ foobnix-3.1.01/debian/source/format
@@ -0,0 +1 @@
+1.0
--- foobnix-3.1.01.orig/debian/watch
+++ foobnix-3.1.01/debian/watch
@@ -0,0 +1,4 @@
+version=3
+
+opts="uversionmangle=s/quantal|precise1|precise|raring|oneiric|natty|vivid1|utopic1|trusty1|l|m//" \
+http://ppa.launchpad.net/foobnix-team/foobnix-player/ubuntu/pool/main/f/foobnix/ foobnix_(.*)\.tar\.gz
