diff -urN stellarium-0.18.2.orig/plugins/TelescopeControl/src/gui/CMakeLists.txt stellarium-0.18.2/plugins/TelescopeControl/src/gui/CMakeLists.txt
--- stellarium-0.18.2.orig/plugins/TelescopeControl/src/gui/CMakeLists.txt	2018-10-18 12:22:13.194365739 +0300
+++ stellarium-0.18.2/plugins/TelescopeControl/src/gui/CMakeLists.txt	2018-10-18 12:23:40.867369065 +0300
@@ -23,6 +23,7 @@
     Qt5::Gui
     TelescopeControl_common
     TelescopeControl_INDI
+    -lQt5SerialPort
     )
 
 SET_TARGET_PROPERTIES(TelescopeControl_gui PROPERTIES FOLDER "plugins/TelescopeControl")
